sdk/core/Azure.Core.TestFramework/src/RecordedTestBase.cs

```diff
@@ -29,6 +30,9 @@ public abstract class RecordedTestBase : ClientTestBase
         public bool SaveDebugRecordingsOnFailure { get; set; } = false;
 #endif
 
+        public JsonWriterOptions RequestSerializationOptions { get; set; } = new JsonWriterOptions();
+        public JsonWriterOptions ResponseSerializationOptions { get; set; } = new JsonWriterOptions();
```

Comment:
I don't see these used currently. And even if they were, shouldn't we have the more permissive encoder by default for the `ResponseSerializationOptions`?